domain/src/test/java/org/oppia/android/domain/classroom/ClassroomControllerTest.kt

```diff
@@ -131,6 +131,36 @@ class ClassroomControllerTest {
     assertThat(classroomList.classroomSummaryList.size).isEqualTo(2)
   }
 
+  @Test
+  fun testGetClassrooms_returnsAllClassrooms() {
+    val classrooms = classroomController.getClassrooms()
+
+    assertThat(classrooms[0].id).isEqualTo(TEST_CLASSROOM_ID_0)
+    assertThat(classrooms[1].id).isEqualTo(TEST_CLASSROOM_ID_1)
+    assertThat(classrooms[2].id).isEqualTo(TEST_CLASSROOM_ID_2)
+  }
+
+  @Test
+  fun testGetClassroomById_returnsFirstClassroom() {
+    val classroom = classroomController.getClassroomById(TEST_CLASSROOM_ID_0)
+
+    assertThat(classroom.id).isEqualTo(TEST_CLASSROOM_ID_0)
+  }
+
+  @Test
+  fun testGetClassroomById_returnsSecondClassroom() {
+    val classroom = classroomController.getClassroomById(TEST_CLASSROOM_ID_1)
+
+    assertThat(classroom.id).isEqualTo(TEST_CLASSROOM_ID_1)
+  }
+
+  @Test
+  fun testGetClassroomById_returnsThirdClassroom() {
+    val classroom = classroomController.getClassroomById(TEST_CLASSROOM_ID_2)
+
+    assertThat(classroom.id).isEqualTo(TEST_CLASSROOM_ID_2)
+  }
```

Comment:
These tests are redudant, since they test the exact same thing. Only one is sufficient in this case. I would suggest insteam testing for the classroom name, since that would return a different outcome for each test.

Reply:
Done.